plugins/WindowManager/TopLevelWindowModel.cpp

```diff
@@ -721,8 +724,29 @@ void TopLevelWindowModel::activateTopMostWindowWithoutId(int forbiddenId)
     }
 }
 
-void TopLevelWindowModel::activateNullWindow()
+void TopLevelWindowModel::rootFocus(bool focus)
+{
+    if (focus) {
+        // Give focus back to previous focused window, only if null window is focused
+        // If null window is not focused, a different app had taken the focus and we
+        // should repect that or if a pendigActivation is going on
```

Comment:
Typo `pendigActivation` here. And missing punctuation (periods). Along with an extra space in `focused,  a different`.